src/main/python/fine_tuning/run_batch.py

```diff
@@ -152,7 +152,7 @@ def del_method_related_files(method_name):
     parser.add_argument('--anserini_root', default='', help='Anserini path')
     parser.add_argument('--collection', required=True, help='the collection key in yaml')
     parser.add_argument('--basemodel', default='bm25', choices=['bm25', 'ql'], help='the ranking model')
-    parser.add_argument('--model', default='axiom', choices=['axiom', 'rm3', 'sdm'], help='the higher level model')
+    parser.add_argument('--model', default='axiom', choices=['bm25', 'ql', 'axiom', 'rm3', 'bm25+axiom', 'bm25+rm3'], help='the higher level model')
```

Comment:
Is it a good idea to mix model with basemodel? I think here is the reason why all your result files start with "bm25"..

Reply:
Or we abandon basemodel all together?

Reply:
Like what you did with `bm+rm3`?